lib/cookie/cookieJar.ts

```diff
@@ -413,22 +418,14 @@ export class CookieJar {
     const store = this.store
 
     if (!store.updateCookie) {
-      store.updateCookie = function (
+      store.updateCookie = async function (
```

Comment:
It's not _required_, as there is no `await`. However, this method always returns a promise, so using an async function is a way to make that contract obvious, even in JS land. And it makes error handling simpler, as an async function will only ever reject errors, rather than throw them.